airbyte-integrations/connectors/source-jira/Dockerfile

```diff
@@ -12,5 +12,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.2.22
+LABEL io.airbyte.version=0.2.23
```

Comment:
@Barsoomx Use 0.2.3 since its a large % of changes

Reply:
0.2.3 already exists. Maybe 0.3.0?

Reply:
@Barsoomx sure 0.3.0 will work